test/components/bst-global-notification.directive.test.js

```diff
@@ -0,0 +1,33 @@
+describe('Directive: bstGlobalNotification', function () {
+    var $scope, $compile, PageTitle;
+
+    beforeEach(module('Bastion.components', 'components/views/bst-global-notification.html', 'components/views/bst-alert.html', 'components/views/bst-alerts.html'));
```

Comment:
Another spacing nitpick but when you have this many modules to include it may make sense to put each on it's own line so:  https://github.com/Katello/katello/blob/master/engines/bastion_katello/test/content-hosts/details/content-host-add-subscriptions.controller.test.js#L13-L20.